tensorwaves/physics/helicity_formalism/amplitude.py

```diff
@@ -509,9 +509,9 @@ def __call__(self, dataset: dict) -> tf.Tensor:
                 dataset[self._params.phi_name],
                 dataset[self._params.theta_name],
                 0.0,
-                2 * self._params.j,
-                2 * self._params.m,
-                2 * self._params.mprime,
+                int(2 * self._params.j),
```

Comment:
Whi is it `int` here, and in in the `_AngularProperties` it's `float`?

Reply:
amplitf works with ints by multiplying everything by 2